docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -151,4 +151,14 @@ Note that this is different to slow operations logged by the Couchbase SDKs - th
 
 *Remediation*: Verify that your Data Service nodes have adequate system resources.
 
+[#CB90052]
+=== Malformed vBuckets (CB90052)
+
+*Background*: Files stored in a Bucket have become corrupted.
+
+*Condition*: vBucket found containing corrupted data.
```

Comment:
NITPICKING: This is a little misleading - "vBucket found" suggests *we* found it, when it was really CBS